content/blog/posts/o3de-introducing-design-system.md

```diff
@@ -0,0 +1,95 @@
+
+These tenets were formed to be straightforward and easy to remember. We want to ensure that the BJDS tenets are flexible for the community to embrace, and to employ through their contributions to the engine.
+
+### What's included in the BlueJay Design System
+
+**The BlueJay Design System** is made up of a set of standardized components and layouts to help accelerate the creation of new workflows for the editor. It also has [supporting documentation](https://o3de.org/docs/tools-ui/) on the O3DE Learn website, which details the following:
+
+#### UI Components
+
+UI components are the various individual widgets that make up the editor experience. The BJDS documentation outlines details like visual markup properties, rules around how to use them and guides when to use them in which scenario. Currently, with [20 UI components](https://o3de.org/docs/tools-ui/component-library/) detailed (and more coming), these include code samples and API references to integrate into your editor extension.
+
+If you are looking for information on key building blocks that O3DE uses, [UI Components](https://o3de.org/docs/tools-ui/component-library/) is the right place to read. It's the systematic reuse of these components that make up the visual and functional consistency across O3DE.
+
+![Preview @mdash; for the O3DE UI component guidelines](/images/blog/intro-to-bjds/ui-components-preview.jpg)
+
+#### UX Patterns
+
+UX Patterns are a series of best practice solutions that a designer or developer should follow when creating new workflows and components in the editor. Unlike UI Components, where it's mostly about the visual aesthetics and how it mechanically operates, UX Patterns is more about how a user interacts and behaves for a task with a goal in mind.
+
+For e.g., the anatomy of a component card and it's different types can be found on the [UI Component > Card page](https://o3de.org/docs/tools-ui/component-library/uidev-card-component/), while the what contents go in the card, how a user interacts with it and how O3DE interacts with the card itself can be found on the [UX Patterns > Component Card UX Patterns](https://o3de.org/docs/tools-ui/ux-patterns/component-card/overview/) page.
+
+![Preview image for the O3DE component card guidelines](/images/blog/intro-to-bjds/componentcard-ux-preview.jpg)
+
+Currently we have two UX Pattern pages on O3DE Learn: Component Card Properties and Writing UI Message guidelines. We plan to roll out more pages in the future coming months, so stay tuned.
+
+#### O3DE Branding Guidelines
+
+Our [brand guidelines section](https://o3de.org/docs/tools-ui/branding-guidelines/) includes information on how to apply the O3DE brand, and downloadable assets such as; a PDF style guide, logos files and fonts etc. These pages are especially important if you are looking to represent the O3DE logo in your product or marketing website. 
```

Comment:
You only need a comma when you're separating two independent clauses, so I removed it before "and downloadable assets..."

```suggestion
Our [brand guidelines section](https://o3de.org/docs/tools-ui/branding-guidelines/) includes information on how to apply the O3DE brand and downloadable assets such as a PDF style guide, logos files, fonts and so on. These pages are especially important if you are looking to represent the O3DE logo in your product or marketing website. 
```